INN/umbrella-inndev
# Lens on Lightfoot Embed - Partner feedback updates

## Changes

This pull request makes the following changes:

- Adds a css conditional to allow elements with `publish-date` to show in the saved link description area, even if the description is not showing, similar to the category button.
- Uses the tag description now as the stories title, and sets `Stories from the project` as the default to be used if no description is present.
- Adds a button to submit story ideas. It hardcodes the beginning portion of the google forms url and adds in the form id if present in the `gform_id` query parameter. If `gform_id` isn't present, the button is not displayed.

Publish date and submit story button:
![Screen Shot 2020-01-23 at 12 10 36 PM](https://user-images.githubusercontent.com/18353636/73006872-62065080-3dd9-11ea-9a38-308164d9fd2c.png)

## Why



For #137

## Testing/Questions

Features that this PR affects:

- Amplify embed feed


Questions that need to be answered before merging:


Steps to test this PR:

1. Update the current tag description to be `Seven independent newsrooms report on Mayor Lori Lightfoot’s first year`. Test it on the embed between this tag and a tag with no description and verify the page title updates accordingly.
2. Add a test published date to a saved link using this, then verify it looks ok on the frontend: 
`<div class="publish-date">Published on January 1, 2020</div>`
3. View the embed and add this query parameter then verify it links ok to the google form:
`gform_id=1FAIpQLSfE8R0oGwgaAcczd_pPdebrlM22RQ_kaewZZKrBEhjyHWtjSA`
4. View the embed with no `gform_id` query parameter and verify the button doesn't appear.
